### DISCUSSION TITLE
Conectar cuenta principal del Facebook.

### DISCUSSION THREAD

**Discussion by RipperxX23** [thatskymod/Sky-CotL-Scripts#144] [Q&A]
Good afternoon. Can the error of connecting the main Facebook account to the canvas be fixed? Because when I connect it tells me that they were working to solve it. In the previous version I was able to connect. Is there a solution?

**Comment by Abbatii** [comment]
I have the same issue here, my account is linked with fb, and the main game won't allow me to link google, Huawei or twitch account. I can't use canvas at all now

**Comment by thatskymod** [comment]
I’m not trying to be difficult, but let me suggest using a different login method. Google, Nintendo, and Local have proven to be reliable.

It's unclear why the Facebook option is still offered when it doesn’t function properly. I understand there may be challenges integrating Sky’s OAuth with a third-party package like Canvas, but that doesn’t justify listing it as an available option. You could raise this with the core [development team](https://github.com/artdeell/Canvas-Open-Source).

In the meantime, switching to another login provider should resolve the issue.
